src/wwPlugin.js

```diff
@@ -212,11 +212,16 @@ export default {
             .find(apiGroup => apiGroup.api === apiGroupUrl);
         if (!apiGroup) return;
 
-        const { data } = await axios.get(apiGroup.swaggerspec, {
-            headers: { Authorization: `Bearer ${this.settings.privateData.apiKey}` },
-        });
+        try {
+            const { data } = await axios.get(apiGroup.swaggerspec, {
+                headers: { Authorization: `Bearer ${this.settings.privateData.apiKey}` },
+            });
 
-        return data;
+            return data;
+        } catch (error) {
+            wwLib.wwLog.error(error);
```

Comment:
can we display the error to the user in the settings?

Reply:
It would be easier to add a message "Only endopoints with swagger enabled will be shown"

Reply:
Ils sont censé corriger ça côté Xano donc je préfère rien mettre comme message, normalement on pourra recup le swagger même désactivé à partir du moment où on a le bearer